### PR TITLE
Few fixes from webOS ports

### PR DESCRIPTION
**Pull request by shr-project**

### CODE REVIEW

**Comment by z3ntu** [comment]
That looks like it fixes #350

File: hybris/tests/test_camera.c [hunk 1]
```diff
@@ -286,6 +292,76 @@ static GLuint create_program(const char* pVertexSource, const char* pFragmentSou
...
+    if (wlcompositor == NULL || wlshell == NULL) {
+	fprintf(stderr, "Can't find compositor or shell\n");
+	exit(1);
+    } else {
+	fprintf(stderr, "Found compositor and shell\n");
+    }
+}
+
+static void
+create_opaque_region() {
```
Comment (mlehtima) [inline hybris/tests/test_camera.c]: Please use consistent function formatting, all old parts had the return value type and function name on same line and the starting curly bracket on the next line. Same goes for other functions you added.

File: hybris/tests/test_camera.c [hunk 2]
```diff
@@ -473,5 +573,10 @@ int main(int argc, char** argv)
 		glDisableVertexAttribArray(gaTexHandle);
 
 		eglSwapBuffers((EGLDisplay) disp, surface);
+		
+		wl_display_dispatch(wldisplay);
 	}
+    
+    wl_display_disconnect(wldisplay);
+
```
Comment (mlehtima) [inline hybris/tests/test_camera.c]: Extra empty line.

File: hybris/tests/test_camera.c [hunk 3]
```diff
@@ -286,6 +292,76 @@ static GLuint create_program(const char* pVertexSource, const char* pFragmentSou
...
+}
+
+static const struct wl_registry_listener registry_listener = {
+    global_registry_handler,
+    global_registry_remover
+};
+
+static void
+get_server_references(void) {
+
```
Comment (mlehtima) [inline hybris/tests/test_camera.c]: Extra empty line.

File: hybris/tests/Makefile.am [hunk 4]
```diff
@@ -140,13 +140,13 @@ test_camera_SOURCES = test_camera.c
 test_camera_CFLAGS = \
 	-I$(top_srcdir)/include \
 	$(ANDROID_HEADERS_CFLAGS)
-
```
Comment (mlehtima) [inline hybris/tests/Makefile.am]: No need to remove this line.

File: hybris/tests/test_camera.c [hunk 5]
```diff
@@ -286,6 +292,76 @@ static GLuint create_program(const char* pVertexSource, const char* pFragmentSou
 	return program;
 }
 
+
+struct wl_display *wldisplay = NULL;
+struct wl_compositor *wlcompositor = NULL;
+struct wl_surface *wlsurface;
```
Comment (mlehtima) [inline hybris/tests/test_camera.c]: Should all of these be initialized to NULL for consistent style?

File: hybris/tests/test_camera.c [hunk 6]
```diff
@@ -286,6 +292,76 @@ static GLuint create_program(const char* pVertexSource, const char* pFragmentSou
...
+    printf("Got a registry event for %s id %d\n", interface, id);
+    if (strcmp(interface, "wl_compositor") == 0) {
+        wlcompositor = wl_registry_bind(registry, 
+				      id, 
+				      &wl_compositor_interface, 
+				      1);
+    } else if (strcmp(interface, "wl_shell") == 0) {
+	wlshell = wl_registry_bind(registry, id,
+				 &wl_shell_interface, 1);
+
```
Comment (mlehtima) [inline hybris/tests/test_camera.c]: Extra empty line.

File: hybris/tests/Makefile.am (hunk 4 above)
Comment (shr-project) [reply hybris/tests/Makefile.am]: Other SOURCES/CFLAGS/LDADD variables are close together without blank lines between, shoudn't this be removed for consistency?

Comment (mlehtima) [reply hybris/tests/Makefile.am]: Looking at other files there is not that much consistency in Makefiles, some have empty lines and others don't. Can be either way.

**Pull request merged**